lib/network/modules/components/shared/ComponentUtil.js

```diff
@@ -1,3 +1,19 @@
+/**
+ * Definitions for param's in jsdoc.
+ * These are more or less global within Network. Putting them here until I can figure out
+ * where to really put them
+ * 
+ * @typedef {string|number} id
+ * @typedef {{x: number, y: number}} point
+ * @typedef {{left: number, top: number, width: number, height: number}} rect
+ * @typedef {{x: number, y:number, angle: number}} rotationPoint
+ *   - point to rotate around and the angle in radians to rotate. angle == 0 means no rotation
+ * @typedef {{nodeId:id}} nodeClickItem
+ * @typedef {{nodeId:id, labelId:id} nodeLabelClickItem
```

Comment:
Missing closing curly

Reply:
Curly! CURLY!!!! Come here, boy!

...damn curly's running away every time. Back to obedience school it is for him.